app/routes/circulars.moderation._index.tsx

```diff
@@ -60,23 +60,16 @@ export async function action({ request }: ActionFunctionArgs) {
 
 export default function () {
   const { changeRequests } = useLoaderData<typeof loader>()
-  const [disableSubmit, setDisableSubmit] = useState(true)
-  const [selectedList, setSelectedList] = useState<string[]>([])
+  const [selectedCount, setSelectedCount] = useState(0)
 
-  function checkboxOnChange(id: string, checked: boolean) {
+  function checkboxOnChange(checked: boolean) {
     if (checked) {
-      setSelectedList([...selectedList, id])
+      setSelectedCount(selectedCount + 1)
     } else {
-      setSelectedList(selectedList.filter((x) => x !== id))
+      setSelectedCount(selectedCount - 1)
     }
```

Comment:
I don't know if Typescript will allow, but due to JavaScript's rules for type promotion, you could do

```ts
setSelectedCount(selectedCount + checked)
```

Elegantly concise? Unholy terror from beyond? You decide.

Also, is there any benefit to using the callback version of setState here? I confess that I do not totally understand when you need that and when you don't.

```ts
setSelectedCount((old) => old + 1)
```